server/athenian/api/controllers/miners/github/pull_request.py

```diff
@@ -35,33 +37,55 @@ def __init__(self, prs: pd.DataFrame, reviews: pd.DataFrame, review_comments: pd
         :param developers: PRs must be authored by these user IDs. An empty list means everybody.
         :param db: Metadata db instance.
         """
-        repos_parsed = [r.split("/") for r in repositories]
-        repo_orgs = [r[0] for r in repos_parsed]
-        repo_names = [r[1] for r in repos_parsed]
         filters = [
             PullRequest.created_at >= time_from,
             PullRequest.created_at < time_to,
-            PullRequest.repository_name.in_(repo_names),
-            PullRequest.repository_owner.in_(repo_orgs),  # FIXME(vmarkovtsev): ENG-101 makes this correct # noqa
+            PullRequest.repository_fullname.in_(repositories),
         ]
         if len(developers) > 0:
             filters.append(PullRequest.user_login.in_(developers))
-        prs = await read_sql_query(select([PullRequest]).where(sql.and_(*filters)),
-                                   db, PullRequest)
-        numbers = prs["number"] if len(prs) > 0 else set()
-        reviews = await read_sql_query(select([PullRequestReview]).where(
-            PullRequestReview.pull_request_number.in_(numbers)), db, PullRequestReview)
-        review_comments = await read_sql_query(select([PullRequestComment]).where(
-            PullRequestComment.pull_request_number.in_(numbers)), db, PullRequestComment)
-        return cls(prs, reviews, review_comments, time_to)
-
-    def __iter__(self) -> Generator[Tuple[pd.Series, pd.DataFrame, pd.DataFrame], None, None]:
+        async with db.connection() as conn:
+            prs = await read_sql_query(select([PullRequest]).where(sql.and_(*filters)),
+                                       conn, PullRequest)
+            numbers = prs[PullRequest.number.key] if len(prs) > 0 else set()
+            reviews = await read_sql_query(select([PullRequestReview]).where(
+                sql.and_(PullRequestReview.pull_request_number.in_(numbers),
+                         PullRequestReview.repository_fullname.in_(repositories))),
+                conn, PullRequestReview)
+            review_comments = await read_sql_query(select([PullRequestComment]).where(
+                sql.and_(PullRequestComment.pull_request_number.in_(numbers),
+                         PullRequestComment.repository_fullname.in_(repositories))),
+                conn, PullRequestComment)
+            commits = await read_sql_query(select([PullRequestCommit]).where(
+                sql.and_(PullRequestCommit.pull_request_number.in_(numbers),
+                         PullRequestCommit.repository_fullname.in_(repositories))),
+                conn, PullRequestCommit)
+        return cls(prs, reviews, review_comments, commits, time_to)
+
+    def __iter__(self) -> Generator[Tuple[pd.Series, pd.DataFrame, pd.DataFrame, pd.DataFrame],
+                                    None, None]:
         """Iterate over the information collected for individual pull requests."""
+        r_repo_key = PullRequest.repository_fullname.key
```

Comment:
I think this should be:
```suggestion
        r_repo_key = PullRequestReview.repository_fullname.key
```